pkg/secretinformer/informer.go

```diff
@@ -0,0 +1,85 @@
+package secretinformer
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+)
+
+const resyncTime = 10 * time.Minute
```

Comment:
Anything behind that value? For the sensor informers we use 1 minute, for things like the admission controller config we use 0 as the resync value.

Reply:
From what I can tell resyncs are essentially used as a fallback retry mechanism. In this case the resync will trigger a call to `updateFn`. I guess it's mostly to recover from cases where the controller misses an event. This is similar to how a watcher disconnects itself after some time requiring an explicit retry.

The guidance I've read is something like "more than a few seconds, less than a few hours" for the period. But there seem to have been some bug fixes in the past, so I'm not sure it's really necessary to resync at all.